bootstrap/cabal-bootstrap-gen.cabal

```diff
@@ -9,10 +9,10 @@ executable cabal-bootstrap-gen
   main-is:          Main.hs
   build-depends:
     , aeson                  ^>=1.5.2.0  || ^>=2.0.3.0 || ^>=2.1.0.0
-    , base                   ^>=4.12.0.0 || ^>=4.13.0.0 || ^>=4.14.0.0 || ^>=4.15.0.0 || ^>=4.16.0.0 || ^>=4.17.0.0
+    , base                   ^>=4.12.0.0 || ^>=4.13.0.0 || ^>=4.14.0.0 || ^>=4.15.0.0 || ^>=4.16.0.0 || ^>=4.17.0.0 || ^>=4.18.0.0
     , bytestring             ^>=0.10.8.2 || ^>=0.11.0.0
-    , Cabal                  ^>=3.4.1.0  || ^>=3.6.3.0 || ^>=3.10.1.0
-    , Cabal-syntax           ^>=3.8.1.0 || ^>=3.10.1.0
+    , Cabal                  ^>=3.4.1.0  || ^>=3.6.3.0 || ^>=3.10.1.0 || ^>=3.12.1.0
+    , Cabal-syntax           ^>=3.8.1.0 || ^>=3.10.1.0 || ^>=3.12.1.0
```

Comment:
The first release from these series will be exceptionally 3.12.0.0, but it's indeed unlikely anybody is bootstrapping until cabal-install is released in 3.12.1.0.